mw-webapp/src/component/table/columns.tsx

```diff
@@ -61,7 +65,7 @@ export const columns: ColumnDef<DayReport, Date & JobDone[] & PlanForNextPeriod[
     cell: (({row}) => {
       return (
         row.original.jobsDone
-          ?.reduce((accum, item) => item.time + accum, 0)
+          ?.reduce((accum, item) => item.time + accum, INITIAL_VALUE)
```

Comment:
INITIAL_VALUE, accum, item - meaningless name, i know it just by signature

Reply:
MIN_VALUE

Reply:
what min value?
no

Reply:
DEFAULT_VALUE

Reply:
DEFAULT_SUMMARY_TIME